docs/pages/getting-started/providers/sailpoint.mdx

```diff
@@ -0,0 +1,255 @@
+import { Callout } from "nextra/components"
+import { Code } from "@/components/Code"
+
+<img align="right" src="/img/providers/sailpoint.svg" height="64" width="64" />
+
+# SailPoint Identity Secure Cloud Provider
+
+SailPoint Identity Secure Cloud (ISC) is an enterprise SaaS platform for identity/security solution. In order to test with this OAuth integration, you will need an ISC tenant, if your are a sailpoint customer or partner, you may inquiry with SailPoint partner/customer success manager. If you are a developer, you may inquiry more information on [SailPoint Developer Community](https://developer.sailpoint.com/discuss/) for more information. 
+
+## Resources
+
+- [SailPoint Identity Secure Cloud Authentication](https://developer.sailpoint.com/docs/api/authentication#choose-authorization-grant-flow)
+- [Managing API Keys and Personal Access Tokens](https://documentation.sailpoint.com/saas/help/common/api_keys.html?h=oauth+client#creating-an-api-key)
+- [SailPoint Developer Community](https://developer.sailpoint.com/discuss/)
+
+## Setup
+
+### Callback URL
+
+<Code>
+  <Code.Next>
+
+```bash
+https://example.com/api/auth/callback/identitySecureCloud
+```
+
+  </Code.Next>
+  <Code.Svelte>
+
+```bash
+https://example.com/api/auth/callback/identitySecureCloud
```

Comment:
In all other packages the default dynamic route is at `/auth/*` not `/api/auth/*`, small difference.

```suggestion
https://example.com/auth/callback/identitySecureCloud
```